web/js/codeworld.js

```diff
@@ -804,6 +804,8 @@ function stop() {
 }
 
 function run(hash, dhash, msg, error, generation) {
+    if (!msg && !error) return;
```

Comment:
What's this for?  It looks like it might result in forgetting to run code that doesn't have warnings?

Reply:
I saw the function ```run``` being executed with these trivial arguments when the page loads up with source code already present. I saw the code was essentially doing nothing, so I placed the return statement at top. 

Should I undo this? Everything works just as well otherwise.
![Screenshot from 2019-03-22 12-52-41](https://user-images.githubusercontent.com/33961252/54806999-d8aa5d00-4ca1-11e9-8fa6-0dfbb537f8e2.png)

Reply:
Yes, I think what you're seeing is a call to the `stop` function.  This may not be necessary on page load, but it's definitely important in other places. I'll merge now and then revert this.

Reply:
Great. Thank you!